tests/RateLimitErrorTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Stripe;
+
+class RateLimitErrorTest extends TestCase
+{
+    private function rateLimitErrorResponse()
+    {
+        return array(
+            'error' => array(),
+        );
+    }
+
+    /**
+     * @expectedException Error\RateLimit
+     */
+    public function testRateLimit()
```

Comment:
But no strong preference, I just find the HTTP code test to be a little "test that PHP runs properly"-ish (though not blatantly so).